server/controllers/logController.js

```diff
@@ -0,0 +1,28 @@
+const Logs = require('../models/logs');
+
+const logController = {};
+
+//store a single log
+logController.storeLog = async (req, res, next) => {
+  try {
+    console.log('storeLog');
+    const { log } = req.body;
+    // TODO: able to store more than one log
+    if (!log) {
+      console.log('missing log');
```

Comment:
You want to use the logger for log the error server side and the error handler to send the information to the client.